packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -319,6 +360,32 @@ export default {
 
         [UPDATE_USER_NOTE]: (state, userNote) => {
             state.userNote = userNote;
+        },
+
+        [UPDATE_FIELD_CHANGES]: (state, field) => {
```

Comment:
This name isn't particularly clear what it's linked to. Does it update the list of changed fields in `state.changes` if one of the fields in the form changes? If so, I'd probably call this `UPDATE_CHANGED_FIELDS` or something similar?

Reply:
updated thanks